### PR TITLE
Jina box

### PR DESCRIPTION
**Pull request by BastinJafari**

### CODE REVIEW

File: chapters/jina_box/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+## Jinabox
+
+##### Jinabox is a lightweight, customizable omnibox. You can use it for searching text, images, videos, audio or any kind of data with a Jina backend.
```
Comment (aga11313) [inline chapters/jina_box/index.md]: Is omnibox a commonly used term? never heard of it before

File: chapters/jina_box/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+![jina box](jinabox.gif)
+
+
+### Features
+
+- Easy-to-use web component for Jina search backend
+- Users can drag and drop multi-modal data as search queries
+- Webcam and microphone support (in progress, contributions welcome! ❤️)
```
Comment (aga11313) [inline chapters/jina_box/index.md]: Are we listing in progress features @FionnD?

File: chapters/jina_box/index.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+![server_endpoint](server_endpoint.png)
+
+### Start Searching!
+
+Drag and drop images, video, or audio, or type in text to search.
+
+![jina box_search](jinabox_search.gif)
+
+
+## Jinabox in your Project
```
Comment (aga11313) [inline chapters/jina_box/index.md:47]: Add a lead up sentence here to make it flow better. Something along the lines of 'this is how you can add a Jina Box to your existing web project'

File: chapters/jina_box/index.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+    <jina-searchbar></jina-searchbar>
+    
+    <!--or-->
+    
+    <jina-floater></jina-searchbar>
+
+Now you can drag anything from anywhere (local/browser/webpage) to it and conduct the search.
+
+Ready to learn Jina? Read our [101 tutorials](../101/index.md).
```
Comment (aga11313) [inline chapters/jina_box/index.md]: its good to have this link but maybe we should put it somewhere else. it feels super out of context here

File: chapters/jina_box/index.md (hunk 2 above)
Comment (FionnD) [reply chapters/jina_box/index.md]: Emm... in general I would not. It's likely we'll forget to update the page in the future and then it's out of sync.

**Review by aga11313 — APPROVED** [review]
LGTM pending all the checks passing :)

**Review by FionnD — APPROVED** [review]

**Pull request merged**